solr/contrib/ltr/src/java/org/apache/solr/ltr/feature/FieldValueFeature.java

```diff
@@ -36,6 +36,10 @@
   private String field;
   private Set<String> fieldAsSet;
 
+
+
+  private double defaultValue = 0.0d;
```

Comment:
Suggest making this a `float` (no need to cast later then but you will need another setter (see StandardNormalizer.java and setAvg for example).